src/Aspire.Hosting/ApplicationModel/ContainerFileSystemCallbackAnnotation.cs

```diff
@@ -53,9 +53,15 @@ public string Name
 public sealed class ContainerFile : ContainerFileSystemItem
 {
     /// <summary>
-    /// The contents of the file. If null, the file will be created as an empty file.
+    /// The contents of the file. Setting Contents is mutually exclusive with <see cref="SourcePath"/>. If both are set, an exception will be thrown.
     /// </summary>
     public string? Contents { get; set; }
+
+    /// <summary>
+    /// The path to a file on the host system to copy into the container. This path must be absolute and point to a file on the host system.
```

Comment:
Why does the path need to be absolute? We can use relative paths elsewhere.

Reply:
We don’t explicitly check for an absolute path; it’s more of a SHOULD requirement. We can’t necessarily guarantee the orchestrator and container runtime will have the correct working directory in all scenarios, so absolute paths guarantees consistent behavior. We probably should call out the same for the bind mount annotation.